pages/common/wormhole.md

```diff
@@ -0,0 +1,23 @@
+# wormhole
+
+> Share files from one computer to another, safely.
+
+- Send a text message:
+
+`wormhole send`
+
+- Send a file/directory:
+
+`wormhole send path/to/file`
+
+- Receive a file/directory/text message:
+
+`wormhole receive access_code
+
+- Receive SSH keys:
+
+`wormhole ssh invite`
+
+- Send SSH keys:
+
+`wormhole ssh accept access_code`
```

Comment:
I believe the access_code here is a parameter. If so, it should be surrounded by two curly brackets.
{{access_code}}